### ISSUE TITLE
upgrading issue with old lib version for reserved field names (class for example)

### ISSUE THREAD

**Issue by ludoch** [EsotericSoftware/yamlbeans#66]
It seems that https://github.com/EsotericSoftware/yamlbeans/commit/5a1db5b3f061c4483e63054c2a73b1d46a156e9e removed the capability of naming the fields independently of the field names in the class...
Not so good, but the bummer is I am upgrading from an old library version, and the published yaml format is using a field named "class" that I cannot obviously uses as class is a reserved keyword in java....
What can I do now?

**Comment by NathanSweet** [comment]
Ideally you could specify a serializer and manually do serialization, like [jsonbeans](https://github.com/EsotericSoftware/jsonbeans). Unfortunately YamlBeans doesn't have a way to do that. Your best bet is likely to modify YamlBeans to do what you need. Eg, add something like:
```
if (fieldName.equals("class")) fieldName = "clazz";
```

**Comment by NathanSweet** [comment]
Sure, that could work. FWIW, YamlBeans is more suited to reading its own output than reading arbitrary data into Java POJOs.

**Comment by NathanSweet** [comment]
Fixed with: d265820333c691551552377dc05170bc1e6bff85
Eg:
```
static public class Meow {
	public String _class;
}

public static void main (String[] args) throws Exception {
	System.out.println(new YamlReader("class: yay").read(Meow.class)._class);
}
```
